source/_lovelace/vertical-stack.markdown

```diff
@@ -10,9 +10,7 @@ sharing: true
 footer: true
 ---
 
-Stack card will allow you to stack together multiple cards so they always sit together in the same column one on top of the other. Keep in mind this can be used with any cards and even used alongside a [horizontal-stack](/lovelace/horizontal-stack/).
-
-## {% linkable_title Options %}
+Vertical stack allows you to group multiple cards to use the same column.
```

Comment:
Reads a little stilted, Maybe *Vertical stack allows you to group multiple cards so they always sit in the same column*?